src/pages/home/ReportScreen.js

```diff
@@ -215,7 +215,10 @@ class ReportScreen extends React.Component {
         const screenWrapperStyle = [styles.appContent, styles.flex1, {marginTop: this.state.viewportOffsetTop}];
 
         // There are no reportActions at all to display and we are still in the process of loading the next set of actions.
-        const isLoadingInitialReportActions = _.isEmpty(this.props.reportActions) && this.props.report.isLoadingReportActions;
+        const isLoadingInitialReportActions = (_.isEmpty(this.props.reportActions) && !this.props.network.isOffline) && this.props.report.isLoadingReportActions;
```

Comment:
Why do we need this change?

Reply:
I guess the thing that confuses me here is why `isLoadingReportActions` would be true if we are offline. I would maybe expect it to be `false` or ask why we are triggering a loader state when we know we can't fetch these actions because we are offline.

Reply:
That was my first question as well. I will look into this part and probably clean it.